src/components/WebPlayBack.tsx

```diff
@@ -1,9 +1,15 @@
-import { useEffect } from 'react';
+import { useEffect, useState} from 'react';
+import PlayerUI, {IPlayerUIProps} from './PlayerUI';
 
 //In current build put Access Token Manually in (Line 18)
 //Look for an issue in Git for authentification for more info.
 function WebPlayback(props: any) {
+    //two state variables for updating Player Object
+    const [statePlayer, setPlayer] = useState();
```

Comment:
minor but I don't like the name statePlayer. Can we make it just player and change the player variable below to be spotifyPlayer or something like that?

Reply:
I am slightly confused, though I agree this is a minor issue. Shouldn't we make it easy for future developers to know that this is related to the state? Is that like implicit? I guess I'm answering my own question I'll make this change. I actually just realized that it is in-fact implicit. Thanks!

Reply:
Actually, I removed const [statePlayer,setPlayer] = useState(); AND setPlayer(player) Becuase we don't need the state information to display the player.

Reply:
Oh good call!